core/src/main/java/org/hibernate/ogm/compensation/impl/InvocationCollectingGridDialect.java

```diff
@@ -60,10 +60,15 @@
  */
 public class InvocationCollectingGridDialect extends ForwardingGridDialect<Serializable> {
 
-	private final EventContextManager eventContext;
```

Comment:
I was expecting this! I'm allergic to cats.

More seriously, any idea on how to fix it without injecting the eventContext later?